Meridian59/Data/Models/AdminData/AdminObjectProperty.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    public class AdminObjectProperty : INotifyPropertyChanged
+    {
+        public string PropertyName { get; set; }
+        public string PropertyType { get; set; }
+        
+        private string _propertyValue { get; set; }
+        public string PropertyValue {
+            get { return _propertyValue; }
+            set
+            {
+                _propertyValue = value;
```

Comment:
You better should check equality of values before setting them when using the INotifyPropertyChanged interface.

Sometimes this can even be important to break out of a 2-way-binding loop:
Assume a user enters a new value into the view, the view updates the model-value by databinding, the model raises a changedevent (2-way-binding), the view updates its value again, which wlll make the model update, which raises changeevent and so on. To break this loop either the view or the model must check for an actually different value. The 'core' can't assume that what's building on top of it is corrrect, so please check in all models implementing INotifyPropertyChanged.